Kickstarter-iOS/Views/Controllers/OptimizelyFeatureFlagToolsViewController.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class OptimizelyFeatureFlagToolsViewController: UITableViewController {
+  // MARK: - Properties
+
+  private var features = OptimizelyFeatures()
```

Comment:
Maybe call this `featuresDataSource`?

Reply:
Hmm so I'm not opposed to this although this isn't subclassing our `ValueCellDataSource` class so that might be misleading. What do you think?